src/wp-content/themes/twentyfourteen/inc/widgets.php

```diff
@@ -67,6 +67,9 @@ public function enqueue_scripts() {
 	 *
 	 * @since Twenty Fourteen 1.0
 	 *
+	 * @global int $content_width Content width.
+	 * @global int $more
```

Comment:
[59101-twentyfifteen.patch](https://core.trac.wordpress.org/attachment/ticket/59101/59101-twentyfifteen.patch) and the Core `register_globals()` function describe the `$more` variable as "Only set, if single page or post." However, that optional description does not make sense to me in the context of the `Twenty_Fourteen_Ephemera_Widget::widget()` method, which sets `$GLOBALS['more'] = 0`.